__tests__/api.test.js

```diff
@@ -99,5 +99,43 @@ describe('API testing', () => {
               expect(body.msg).toBe("Bad Request");
             });
         });
-      });
+    });
+    describe("GET /api/articles/:article_id/comments", () => {
+        test("returns an array of category objects", () => {
+            return request(app)
+            .get("/api/articles/5/comments")
+            .expect(200)
+            .then(({ body }) => {
+                const { comments } = body;
+                expect(comments).toHaveLength(2)
+                comments.forEach((comment) => {
+                expect(comment).toEqual(
+                    expect.objectContaining({
+                    body: expect.any(String),
+                    author: expect.any(String),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    comment_id: expect.any(Number),
+                    })
+                );
+                });
+            });
+        });
```

Comment:
Please add another test where the article ID exists but there are no comments. Expecting an empty array and status 200.